File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.py
def solution(s):
    return True if len(s) in (4,6) and s.isdigit() else False

'''
def solution(s):
    for i in s:
        if i=='0' or i=='1' or i=='2' or i=='3'or i=='4' or i=='5' or i=='6' or i=='7' or i=='8' or i=='9':
            continue
        else:
            return False
    return True
'''

File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.java
class Solution {
    public boolean solution(String s) {
        boolean answer = true;
        //문자열 길이 판단
        if(s.length()==4 || s.length()==6){
            for(int i=0;i<s.length();i++){
                char ch=s.charAt(i);
                if(ch=='0'||ch=='1'||ch=='2'||ch=='3'||ch=='4'||ch=='5'||ch=='6'||ch=='7'||ch=='8'||ch=='9'){
                }
                else{
                    return false;
                }
                System.out.println();
            }
            
        }else{
            return false;
        }
        return answer;
    }
}
